Lib/_pyrepl/unix_console.py

```diff
@@ -390,7 +391,15 @@ def restore(self):
             os.write(self.output_fd, b"\033[?7h")
 
         if hasattr(self, "old_sigwinch"):
-            signal.signal(signal.SIGWINCH, self.old_sigwinch)
+            if os.name != 'nt':
```

Comment:
Oh actually I'm sorry, but since it's UnixConsole, we shouldn't be able to be on Windows. So this additional `os.name != 'nt'` should not be necessary.

Reply:
ok will change that thank you 
seems my first change plus second diff is right, learn a lot, thanks